File: uri/bonus/bonus.pas
program bonus(Input,Output);

uses
    SysUtils;

function calculasalario( var salario: Double; var montante: Double) : Double;
var 
    poo : Double;

begin
    poo := (montante * 15.0) / 100.0;
    calculasalario := salario + poo;
end;

Var 
    nome : String;
    salario, montante, total: Double;

begin
    readLn(nome);
    readLn(salario);
    readLn(montante);
    total := calculasalario( salario, montante );
    writeln('TOTAL = R$ ', Format('%2f',[total]));

end.

File: uri/bonus/bonus.py
#*********************************** Python 2 **************************************

def calculabonus( salario, montante):
    return salario + ((montante * 15)/100)

nome = raw_input()
salario = float(input())
montante = float(input())
total = calculabonus(salario, montante)
print "TOTAL = R$ {0:.2f}".format(total)

#*********************************** Python 3 ************************************

def calculabonus( salario, montante):
    return salario + ((montante * 15)/100)

nome = input()
salario = float(input())
montante = float(input())
total = calculabonus(salario, montante)
print ("TOTAL = R$ {0:.2f}".format(total))